config/oracle_jre.yml

```diff
@@ -20,7 +20,7 @@
 # e.g.  repository_root: "http://example.com/oracle-jre/{platform}/{architecture}"
 ---
 jre:
-  version: 1.8.0_+
+  version: 1.8.0_12+
```

Comment:
We'll have to update it sadly, the matcher the build pack is using isn't that great. I am going to put it as a note in the docs for getting the jre/jdks in our AWS bucket.